project.php

```diff
@@ -407,20 +407,6 @@ function do_project_info_table()
     // -------------------------------------------------------------------------
     // Information about the work itself (independent of DP)
 
-    // the array below should guarantee that the strings 'beginner',
```

Comment:
Actually, I need to update this to use the new function to get the translated values. Will do that soon and update this PR.

Reply:
Fixed.